### PR TITLE
HiveFooter

### PR DESCRIPTION
**Pull request by hasparus**
Added a new footer to use between all landing pages under Hive GraphQL Management Platform umbrella.

image

image

image

### CODE REVIEW

File: packages/components/tsup.config.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import * as fs from 'fs/promises';
```
Comment (saihaj) [inline packages/components/tsup.config.ts:1]: why add?
```suggestion
```

**Review by saihaj — CHANGES_REQUESTED** [review]
Looking at PR description image there should be more spacing so something feels not right

CleanShot 2024-08-30 at 10 48 13@2x

File: packages/components/tsup.config.ts (hunk 1 above)
Comment (hasparus) [reply packages/components/tsup.config.ts:1]: Accident. Thanks for catching it.
I was thinking about moving the fonts here, and copying them to `dist`, and this is a leftover.

Just removed it.

**Comment by hasparus** [comment]
@saihaj I've made some fixes to the footer on the Navigation branch (as it includes the fixed dark mode handling).

https://github.com/the-guild-org/docs/pull/1638/files#diff-97a509d8b04cf4965cbfe27f9a160cb13d568fd7e6d58b56ae15e42b7b5a95bf

Would you like me to bring them over here or you fine with me merging those PRs in a batch?

**Review by saihaj — APPROVED** [review]
I see @hasparus let's merge this one

**Pull request merged**